app/assets/javascripts/components/common/ArticleViewer/containers/ArticleViewer.jsx

```diff
@@ -27,6 +27,7 @@ import colors from '@components/common/ArticleViewer/constants/colors';
 
 // Actions
 import { resetNeedHelpAlert, submitBadWorkAlert } from '~/app/assets/javascripts/actions/alert_actions.js';
+import { Button } from 'react-scroll';
```

Comment:
What is this for?

Reply:
I forgot to remove it. I will remove it on the next commit.